app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  module V1
+    # User Controller
+    class UsersController < ApplicationController
+      before_action :set_user, only: %i[show update destroy]
+
+      # GET /users/1
+      def show
+        render json: User.find(params[:id])
+      end
+
+      # POST /users
+      def create
+        @user = User.new(user_params)
```

Comment:
I don't think you need this to be a instance variable.
```suggestion
        user = User.new(user_params)
```